dockerfiles/mingw-fedora/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM fedora:30
+
+# Update base system
+RUN dnf update -y
+
+# Add some useful packages to the base system
+RUN dnf install -y \
+        make \
+        cmake \
+        git \
+        sudo \
+        wget \
+        gawk \
+        sudo \
+        fakeroot \
+        file \
+        patch \
+        ninja-build
+
+# Install MingW packages
+RUN dnf install -y \
+        mingw32-binutils \
+        mingw32-boost \
+#        mingw32-cmake \
+        mingw32-crt \
+        mingw32-curl \
+        mingw32-fontconfig \
+        mingw32-freetype \
+        mingw32-gcc \
+        mingw32-headers \
+        mingw32-jansson \
+        mingw32-libpng \
+        mingw32-libzip \
+        mingw32-openssl \
+        mingw32-pkg-config \
+        mingw32-SDL2 \
+        mingw32-SDL2_ttf \
+        mingw32-speexdsp \
+        mingw-w64-tools \
+        mingw32-winpthreads \
+        mingw32-win-iconv \
```

Comment:
I think we don't use iconv on Windows, what is it required for?

Reply:
I don't know if iconv is being used or not. It was in the mingw-arch list of packages, so I got the fedora version of the package.